core/scripts/common/helpers.go

```diff
@@ -408,39 +419,59 @@ func GetRlpHeaders(env Environment, blockNumbers []*big.Int) (headers [][]byte,
 				new(big.Int).Set(blockNum).Add(blockNum, big.NewInt(1)),
 			)
 			if err != nil {
-				return nil, fmt.Errorf("failed to get header: %+v", err)
+				return nil, hashes, fmt.Errorf("failed to get header: %+v", err)
 			}
 			// We can still use vanilla go-ethereum rlp.EncodeToBytes, see e.g
 			// https://github.com/ava-labs/coreth/blob/e3ca41bf5295a9a7ca1aeaf29d541fcbb94f79b1/core/types/hashing.go#L49-L57.
-			rlpHeader, err := rlp.EncodeToBytes(h)
+			rlpHeader, err = rlp.EncodeToBytes(h)
 			if err != nil {
-				return nil, fmt.Errorf("failed to encode rlp: %+v", err)
+				return nil, hashes, fmt.Errorf("failed to encode rlp: %+v", err)
 			}
 
+			if hashes != nil && len(hashes) > i {
```

Comment:
Nil check is not necessary here, can just do `hashes = append(hashes, h.Hash())`

Reply:
Yep, I'd originally sent in hashes instead of returning it. Will fix it now